resolwe_bio/tools/expression2storage.py

```diff
@@ -30,12 +33,16 @@ def isfloat(value):
     except ValueError:
         return False
 
-with utils.gzopen(fname) as f:
+with utils.gzopen(args.input) as f:
     # Split lines by tabs
     # Ignore lines without a number in second column
     # Build a dictionary of gene-expression pairs
     exp = {'genes': {utils.escape_mongokey(gene_exp[0]): float(gene_exp[1]) for
                      gene_exp in (l.split('\t') for l in f) if
                      len(gene_exp) == 2 and isfloat(gene_exp[1])}}
 
-print('{"exp_json":%s}' % json.dumps(exp, separators=(',', ':')))
+if args.output:
+    with open(args.output, 'w') as file:
```

Comment:
Do not use `file`, use `f` instead. Strange that pylint does not report this as error, maybe because of Python 3.